tests/integration/test_manylinux.py

```diff
@@ -1000,8 +999,14 @@ def test_build_wheel_compat(
             repaired_tag = f"{expect_tag}.{target_tag}"
         repaired_wheel = f"testsimple-0.0.1-{PYTHON_ABI}-{repaired_tag}.whl"
         assert repaired_wheel in filenames
+
         assert_show_output(manylinux_ctr, repaired_wheel, expect, True)
 
+        with zipfile.ZipFile(os.path.join(io_folder, repaired_wheel)) as z:
+            for file in z.namelist():
+                if file.startswith("testsimple.libs"):
+                    raise Exception("should not have empty .libs folder")
```

Comment:
```suggestion
                assert not file.startswith("testsimple.libs"), "should not have empty .libs folder"
```

Reply:
thanks, forget there is an assert statment...